marker/config/parser.py

```diff
@@ -53,23 +49,17 @@ def generate_config_dict(self) -> Dict[str, any]:
                     config["debug_data_folder"] = output_dir
                 case "page_range":
                     config["page_range"] = parse_range_str(v)
-                case "force_ocr":
-                    config["force_ocr"] = True
                 case "languages":
                     config["languages"] = v.split(",")
                 case "config_json":
                     with open(v, "r") as f:
                         config.update(json.load(f))
                 case "disable_multiprocessing":
                     config["pdftext_workers"] = 1
-                case "paginate_output":
-                    config["paginate_output"] = True
                 case "disable_image_extraction":
                     config["extract_images"] = False
-                case "use_llm":
-                    config["use_llm"] = True
-                case "strip_existing_ocr":
-                    config["strip_existing_ocr"] = True
+                case _:
+                    config[k] = v
```

Comment:
We should probably check to see if the config key is valid here, now that we have an easy way to do that

Reply:
done!